src/istio/control/http/attributes_builder_test.cc

```diff
@@ -465,6 +470,8 @@ TEST(AttributesBuilderTest, TestReportAttributes) {
   Attributes expected_attributes;
   ASSERT_TRUE(
       TextFormat::ParseFromString(kReportAttributes, &expected_attributes));
+  (*expected_attributes.mutable_attributes())[AttributeName::kDestinationUID]
+      .set_string_value("pod1.ns2");
```

Comment:
can we just add this attribute to kReportAttributes?

Reply:
There's unexpected use of the kReportAttributes in auth tests. This keeps the change contained to the tests of Report.